yacc.go

```diff
@@ -3470,22 +3469,20 @@ func $$Parse($$lex $$Lexer) int {
 
 func ($$rcvr *$$ParserImpl) Parse($$lex $$Lexer) int {
 	var $$n int
-	var $$lval $$SymType
```

Comment:
Eh? It passes `lval` to our lexer. I can't see how escape analysis could detect the value doesn't eventually escape here.